### PR TITLE
Added visualisation for time taken for each activity

### PR DESCRIPTION
**Pull request by bensjx**
Access via /fusionchartdemo

### CODE REVIEW

File: src/containers/Debug/.vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (FeynmanDNA) [inline src/containers/Debug/.vscode/launch.json]: @bensjx hi, what does this launch.json file do?

File: src/containers/Debug/fusionChartDemo.js [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+import FusionTheme from 'fusioncharts/themes/fusioncharts.theme.fusion';
+
+import Button from '@material-ui/core/Button';
+import { Typography } from "@material-ui/core";
+
+ReactFC.fcRoot(FusionCharts, Charts,powerCharts, FusionTheme);
+
+
+class fusionChartDemo extends React.PureComponent {
+    constructor(props) {
```
Comment (FeynmanDNA) [inline src/containers/Debug/fusionChartDemo.js]: hi, three suggestions:
1. React class already provide the `props`, and you can initialize the state by calling `state = {}` as a class property.
2. Can you follow the indentation of 2 spaces like the rest of the codebase
3. This class component can be in a different `container` folder, on the same level as `Debug`

File: src/containers/Debug/fusionChartDemo.js [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+    type: 'boxandwhisker2d',
+    width: '100%',
+    height: 500,
+    dataFormat: 'json',
+    dataSource: myDataSource,
+};  
+
+
+const mapStateToProps = (state, ownProps) => ({
+    uid: state.firebase.auth.uid,
```
Comment (FeynmanDNA) [inline src/containers/Debug/fusionChartDemo.js]: do you need these states from the redux store in this component?

**Comment by FeynmanDNA** [comment]
@bensjx I have grouped the demos and ideas being tried to a new container folder `IdeaLab` in `containers/`.

You can create a new folder in `IdeaLab` for your trial project after you pulled from the lastest code.

File: src/containers/Debug/.vscode/launch.json (hunk 1 above)
Comment (bensjx) [reply src/containers/Debug/.vscode/launch.json]: Hi, this file is not needed, i will be deleting it

File: src/containers/Debug/fusionChartDemo.js (hunk 2 above)
Comment (bensjx) [reply src/containers/Debug/fusionChartDemo.js]: Ok, noted!

File: src/containers/Debug/fusionChartDemo.js (hunk 3 above)
Comment (bensjx) [reply src/containers/Debug/fusionChartDemo.js]: I don't think so. I will be making the changes

**Comment by FeynmanDNA** [comment]
@bensjx it is better to finalize the local changes before triggering another Travis CI service. like creating a new branch

**Comment by bensjx** [comment]
> @bensjx it is better to finalize the local changes before triggering another Travis CI service. like creating a new branch

Alright, understood :)

**Pull request merged**